### DISCUSSION TITLE
How to support gap for ios < 14.5

### DISCUSSION THREAD

**Discussion by quangnv13** [tailwindlabs/tailwindcss#6969] [Help]
I'm using tailwindcss 3 and it's work perfectly with postcss + autoprefixer. But only ios < 14.5 browsers gap is not supported. If i used gap utility i need to auto convert gap to margin or anything can support for ios < 14.5.

Someone can't help me.

**Comment by quangnv13** [comment]
Someone have solution for this?

**Comment by quangnv13** [comment]
I think space between will resolve my problem

**Comment by trangchongcheng** [comment]
Yes, I have same issue, gap property is not support for ios 14.1 (with me), so you can switch to using space-x or space-y instead of.
